modules/k8s/Makefile.tanka

```diff
@@ -28,3 +28,6 @@ k8s/tanka/delete/%:
 	@app="$*"; \
 	tk delete environments/$$app/local
 
+## Creates a new app in the existing repo
+k8s/tanka/new-app/%:
```

Comment:
I wonder if there's anything else we would do to an app, if so the following might be more future-proof:
```
k8s/tanka/app/<name>/new|add
k8s/tanka/app/<name>/delete|remove
````

Reply:
true, you never know :)

will change it to 'add' for now